openedx/core/djangoapps/content_libraries/rest_api/serializers.py

```diff
@@ -158,6 +159,7 @@ class LibraryXBlockMetadataSerializer(serializers.Serializer):
     tags_count = serializers.IntegerField(read_only=True)
 
     collections = CollectionMetadataSerializer(many=True, required=False)
+    can_stand_alone = serializers.BooleanField(read_only=True)
```

Comment:
ditto here -- I don't think we should expose `can_stand_alone` to serializers either.